lib/internal/events/abort_listener.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
+
+const {
+  validateAbortSignal,
+  validateFunction,
+} = require('internal/validators');
+const {
+  codes: {
+    ERR_INVALID_ARG_TYPE,
+  },
+} = require('internal/errors');
+const { SymbolDispose } = primordials;
```

Comment:
```suggestion
const {
  SymbolDispose,
} = primordials;
const {
  validateAbortSignal,
  validateFunction,
} = require('internal/validators');
const {
  codes: {
    ERR_INVALID_ARG_TYPE,
  },
} = require('internal/errors');
```

Reply:
@aduh95 Happy to change, just curios why 🙂
Is this is a documented convention?

Reply:
Most files have `primordials` at the very top of the file, I don't know if it's documented, but it seems good to have it there for the sake of consistency :)